src/backends/tensorrt/plan_backend.cc

```diff
@@ -170,6 +170,14 @@ PlanBackend::Context::Context(
     events_[idx].timestamp_signal_ = nullptr;
   }
   support_batching_ = (max_batch_size != NO_BATCHING);
+
+  Status status = SupportsIntegratedZeroCopy(gpu_device_, &zero_copy_support_);
+  if ((!status.IsOk()) || (!zero_copy_support_)) {
```

Comment:
I thought you said you were going to print the error message if there is an error?

Reply:
Ah right. Fixed now.